code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -343,6 +344,7 @@
 	righthand_file = 'icons/mob/inhands/misc/food_righthand.dmi'
 	container_type = NONE
 	spillable = FALSE
+	you_drink_from_this = TRUE
```

Comment:
Redundant. You already set this variable to true on the parent class of `/obj/item/reagent_containers/food/drinks`

Reply:
For some reason, it only worked when I put it in both of them